### PR TITLE
Xen mitigation adaption for hvm on sle15sp6

### PR DESCRIPTION
**Pull request by museQi**
Xen mitigation adaption for hvm on sle15sp6,because of the changes
- Related ticket: https://progress.opensuse.org/issues/151352
- Verification run:http://10.67.129.4/tests/73044;http://10.67.129.4/tests/73113;http://10.67.129.4/tests/73112

### CODE REVIEW

**Comment by museQi** [comment]
@XGWang0 @JoyceNa @zyuhu Please help me review.

**Comment by XGWang0** [comment]
LGTM

File: tests/cpu_bugs/xen_domu_mitigation_test.pm [hunk 1]
```diff
@@ -131,7 +131,13 @@ my $pti_on_on_hvm = {"pti=on" => {
         }
     }
 };
-
+my $pti_on_on_hvm_for_cascadelake = {"pti=on" => {
+        default => {
+            expected => {'cat /proc/cmdline' => ['pti=on'], 'cat /sys/devices/system/cpu/vulnerabilities/meltdown' => ['Not affected']},
+            unexpected => {'cat /proc/cmdline' => ['pti=off'], 'cat /sys/devices/system/cpu/vulnerabilities/meltdown' => ['Unknown.*XEN PV detected,.*hypervisor mitigation required']}
```
Comment (zyuhu) [inline tests/cpu_bugs/xen_domu_mitigation_test.pm:137]: I have a stupid question, for these logic, the expected value is fixed, however, is the unexpected value also fixed? or not? So are all below unexpected values logic.

Comment (museQi) [reply tests/cpu_bugs/xen_domu_mitigation_test.pm:137]: unexpected value not fixed. The secnarion is when come across the expection string,If it doesn't appear go die.
when come across the unexpect string,if it appears go die. And the default match is the expect. So I did not fix the unexpected value .

**Comment by museQi** [comment]
Because of the duplicate as the pr https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19895.So I close it.

**Pull request closed without merging**